device-types/Blackmagicdesign/ATEM-2ME-Production-Studio-4K.yaml

```diff
@@ -0,0 +1,123 @@
+---
+manufacturer: Blackmagicdesign
```

Comment:
I suggest we name this manufacturer `Blackmagic Design` instead

Reply:
Jeg er egentlig enig, men dette formatet var det netbox-device-greiene ville ha.